features/naam/naam.feature

```diff
@@ -1,178 +1,173 @@
-      | GI                      | gravin                     | titel               | O                   | onbekend              |
-      | H                       | hertog                     | titel               | M                   | man                   |
-      | HI                      | hertogin                   | titel               | V                   | vrouw                 |
-      | M                       | markies                    | titel               | M                   | man                   |
-      | MI                      | markiezin                  | titel               | V                   | vrouw                 |
-      | P                       | prins                      | titel               | M                   | man                   |
-      | PS                      | prinses                    | titel               | V                   | vrouw                 |
-      | R                       | ridder                     | titel               | M                   | man                   |
-      | R                       | ridder                     | titel               | V                   | vrouw                 |
-      | JH                      | jonkheer                   | predicaat           | M                   | man                   |
-      | JV                      | jonkvrouw                  | predicaat           | V                   | vrouw                 |
\ No newline at end of file
+  Scenario: naam van mogelijke bewoner wordt geleverd bij het raadplegen van bewoning met peildatum in onzekerheidsperiode
+    Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+    | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) |
+    | Robin Sam         | van den             | Aedel                 |
+    En de persoon is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100000                           |
+    Als bewoningen wordt gezocht met de volgende parameters
+    | naam                             | waarde               |
+    | type                             | BewoningMetPeildatum |
+    | peildatum                        | 2010-09-01           |
+    | adresseerbaarObjectIdentificatie | 0800000000000001     |
+    En heeft de response een bewoning met de volgende gegevens
+    | naam                             | waarde                    |
+    | periode                          | 2010-09-01 tot 2010-09-02 |
+    | adresseerbaarObjectIdentificatie | 0800000000000001          |
+    En heeft de bewoning een mogelijke bewoner met de volgende gegevens
+    | burgerservicenummer | naam.voornamen | naam.voorvoegsel | naam.geslachtsnaam | 
+    | 000000024           | Robin Sam      | van den          | Aedel              |
+
+Regel: Voornamen/geslachtsnaam met diakrieten wordt geleverd
```

Comment:
Naar mijn mening een onduidelijke regel. Wat wordt bedoeld met `als voornamen/geslachtsnaam diakrieten bevat zijn beide gevuld`? Ik denk dat het volgende wordt bedoeld:

```
Als een (mogelijke) bewoner ook voornamen en/of geslachtsnaam met diakrieten heeft, dan wordt deze geleverd
```

Reply:
Het gaat er hier om dat het in de praktijk niet kan voorkomen dat de kolom `voor_namen_diak` een waarde bevat en de kolom `voor_namen` leeg is. Hetzelfde geld voor `geslachts_naam_diak` en `geslachts_naam`. De term 'beide' in deze regel slaat enerzijds op de velden `voor_namen` en `voor_namen_diak` en anderzijds op de velden `geslachts_naam` en `geslachts_naam_diak`.

Andersom kan wel voorkomen. Namelijk, als de voornamen of geslachtsnaam geen diakrieten bevatten heeft de kolom `voor_namen` wel een waarde en `voor_namen_diak` niet. Of, `geslachts_naam` bevat een naam en `geslachts_naam_diak` is leeg. Dit is toegestaan.

Dit scenario laat zien dat de waarde in kolom `voor_namen` geleverd wordt als de kolom `voor_namen_diak` geen waarde bevat en de als de (mogelijke) bewoner een voornaam met diakrieten heeft de waarde in kolom `voor_namen_diak` geleverd wordt. 

De regel is aangepast naar: 

```
Als een (mogelijke) bewoner ook voornamen en/of geslachtsnaam met diakrieten heeft, dan wordt deze geleverd:
    - als voornamen/geslachtsnaam diakrieten bevat zijn beide gevuld. De velden bevatten verschillende waarden.
    - als voornamen/geslachtsnaam geen diakrieten bevat is alleen voornamen (02.10)/geslachtsnaam (02.40) gevuld.
```